castervoice/core/controller.py

```diff
@@ -1,13 +1,21 @@
 import logging
+import os
 import yaml
 
+try:
+    import importlib.resources as pkg_resources
+except ImportError:
+    # Try backported to PY<37 `importlib_resources`.
+    import importlib_resources as pkg_resources
```

Comment:
You could use [importlib-resources](https://pypi.org/project/importlib-resources/)

Reply:
I've added importlib-resources now..

Reply:
And will remove it again to drop support for v3.6